dask/array/tests/test_reductions.py

```diff
@@ -473,30 +473,30 @@ def test_topk_argtopk1(npfunc, daskfunc, split_every):
 
     # 1-dimensional arrays
     # top 5 elements, sorted descending
-    assert_eq(npfunc(a)[-k:][::-1],
+    assert_eq(npfunc(a.compute())[-k:][::-1],
```

Comment:
A small, but non-critical comment. Would it make sense to actually store the NumPy arrays above and just use them down here as opposed to calling `.compute()` repeatedly?

Reply:
Yes, you're right. I'm gonna fix that.